rfq/oracle.go

```diff
@@ -19,6 +20,19 @@ type OracleError struct {
 	Msg string
 }
 
+// String returns a human-readable string representation of the error.
+func (o *OracleError) String() string {
```

Comment:
Doesn't a custom error need a `Error()` method that returns a string in order to satisfy the `error` interface? Maybe we should do that instead?